modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -119,6 +122,9 @@ GrainTracker::finalize()
                                                 _unique_grain_to_ebsd_num[grain_pair.first] :
                                                 grain_pair.first,
                                                 grain_pair.second._var_idx);
+
+          auto elem_it_pair = _elemental_data_2.emplace(elem_id, std::vector<unsigned int>(_n_vars, 0));
```

Comment:
This is nice!

Did you see this one: http://en.cppreference.com/w/cpp/container/map/emplace_hint

Could be awesome if you're putting lots of things into the map in increasing order...

`try_emplace` is also really cool looking: http://en.cppreference.com/w/cpp/container/map/try_emplace